app/pages/project/talk-images.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import getSubjectLocation from '../../lib/get-subject-location.coffee';
+import TalkStatus from './talk-status';
+import VideoPlayer from '../../components/video-player';
+
+export default class TalkImages extends React.Component {
+
+  render() {
+    return (
+      <div className="project-home-page__container">
+
+        {this.props.images.map((image) => {
+          const subject = getSubjectLocation(image);
+          if (subject.type === 'video') {
+            return (
+              <div key={image.id} className="project-home-page__talk-image">
+                <VideoPlayer src={subject.src} type={subject.type} format={subject.format} showControls={false} />
+              </div>
+            );
+          }
+          return (
+            <div key={subject.id} className="project-home-page__talk-image">
+              <img alt="" src={subject.src} />
```

Comment:
The Thumbnail component will also give you a small, cropped jpeg, so that you aren't downloading large subject images and resizing in the browser.